src/main/resources/graphdb-kafka-sink.properties

```diff
@@ -15,6 +15,7 @@
 
 name=GraphDBSinkConnector
 connector.class=com.ontotext.kafka.GraphDBSinkConnector
+connector.client.config.override.policy=All
```

Comment:
Can the readme mention what this does?

Reply:
Yes, but with the new connect version it has to be set up explicitly, not sure why. I can include some info in the readme.